mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -30,8 +30,11 @@ import org.apache.spark.sql.{DataFrame, Row}
 
 
 /**
+ * :: Experimental ::
  * Common params for KMeans and KMeansModel
  */
+@Since("1.5.0")
+@Experimental
```

Comment:
Both `Since` and `Experimental` are not necessary because this is package private.